ci-operator/step-registry/hypershift/mce/install/disconnected/hypershift-mce-install-disconnected-commands.sh

```diff
@@ -265,4 +265,6 @@ oc wait --timeout=5m --for=condition=ManagedClusterJoined -n local-cluster Manag
 echo "MCE local-cluster is ready!"
 
 set -x
-EOF
\ No newline at end of file
+EOF
+
+oc get imagecontentsourcepolicy -o json | jq -r '.items[].spec.repositoryDigestMirrors[0].mirrors[0]' > "${SHARED_DIR}/mirror_registry_url"
```

Comment:
This step updates this `mirror_registry_url` file incorrectly, https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/43339/rehearse-43339-periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-agent-ovn-mce-disconnected-conformance/1712350507655237632/artifacts/e2e-agent-ovn-mce-disconnected-conformance/enable-qe-catalogsource-disconnected/build-log.txt